adapters/Mintegral/MintegralAdapter/Bidding/GADMAdapterMintegralBannerLoader.m

```diff
@@ -0,0 +1,158 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "GADMAdapterMintegralBannerLoader.h"
+#import "GADMAdapterMintegralUtils.h"
+#import "GADMediationAdapterMintegralConstants.h"
+
+#include <stdatomic.h>
+#import <MTGSDK/MTGSDK.h>
+#import <MTGSDKBanner/MTGBannerAdView.h>
+#import <MTGSDKBanner/MTGBannerAdViewDelegate.h>
+
+static CGSize const mintegralBannerAdSize320x50 = (CGSize){320, 50};
+static CGSize const mintegralBannerAdSize320x100 = (CGSize){320, 100};
+static CGSize const mintegralBannerAdSize300x250 = (CGSize){300, 250};
+static CGSize const mintegralBannerAdSize728x90 = (CGSize){728, 90};
+
+@interface GADMAdapterMintegralBannerLoader ()
+<GADMediationBannerAd,
+MTGBannerAdViewDelegate>
```

Comment:
Can you move these to the `h` file, the same applies to the other files.

Reply:
thanks, i have fixed it.